probe_scraper/parsers/repositories.py

```diff
@@ -144,6 +144,7 @@ def parse_v2(self, filename=None) -> dict:
                     "additional_dependencies", []
                 )
                 listing = {**app, **channel}
+                listing["canonical_app_description"] = app["description"]
```

Comment:
It sounds good to distinguish `canonical_app_description` from description. I think we should update the content of `repositories.yaml` to rename the app-level "description" field to "canonical_app_description" to make this more explicit and avoid having to do this manipulation here.